superset/views/core.py

```diff
@@ -2459,8 +2459,9 @@ def cache_key_exist(self, key):
 
     @has_access_api
     @expose("/results/<key>/")
+    @expose("/results/<key>/<int:rows>")
```

Comment:
it seems a little weird to encode the limit in the url like this. I think it would be preferable to construct the url like: `/results/key?rows=1000`.

Reply:
+1